app/controllers/organizations/adopter_fosterer/adopted_pets/files_controller.rb

```diff
@@ -7,14 +7,7 @@ class FilesController < Organizations::BaseController
         def index
           @pet = Pet.find(params[:adopted_pet_id])
           respond_to do |format|
-            format.html # Regular HTML response
-            format.turbo_stream do
-              render turbo_stream: turbo_stream.replace(
-                "pet_files",
-                partial: "organizations/shared/file_attachment_table",
-                locals: {pet: @pet}
-              )
-            end
+            format.html
```

Comment:
I think we can remove this as it’s the default format?

Reply:
Done